__tests__/app.test.js

```diff
@@ -44,8 +44,52 @@ describe("GET /api", () => {
     return request(app).get("/api").expect(200);
   });
   it("should contain the same information as endpoints.json", () => {
-    return request(app).get("/api").expect(200)
-    .then(data => expect(data.body).toEqual(endPoints));
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((data) => expect(data.body).toEqual(endPoints));
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  it("responds with a 200 status message", () => {
+    return request(app).get("/api/articles/2").expect(200);
+  });
+  it("responds with an object which contains article information", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then(({ body }) =>
+        expect(body).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
```

Comment:
Nice use of toMatchObject here 👍 make sure that you are asserting for the actual number of ID in the tests - you are expecting 2 here 😄